tools/cli/clitest/context.go

```diff
@@ -61,6 +61,15 @@ func BoolArgument(name string, value bool) CliArgument {
 	}
 }
 
+// Int64Argument introduces a new int64 argument for cli context
+func Int64Argument(name string, value int64) CliArgument {
```

Comment:
Maybe we could just do this with generics?

Reply:
Hmm... that's true. At the same time, lots of those are just const-s, and then you have to convert them to types explicitly anyway.